src/main/java/no/fk/fint/controller/AnsattController.java

```diff
@@ -29,6 +29,7 @@
 
 @Slf4j
 @RestController
+@CrossOrigin(origins = "http://localhost:9000")
```

Comment:
Kan vi gjøre denne mer generell?
Jeg mener å huske at vi har brukt et filter her som setter på header for allow-origin, slik at CORS er i orden.